File: Java/ForEncadeado.java
import java.util.Locale;
import java.util.Scanner;

public class ForEncadeado {
    public static Scanner scanf = new Scanner(System.in);
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);

        System.out.println("Digite um numero inteiro: ");
        int numeroX = scanf.nextInt();

        for(int i = 1; i <= numeroX; i++){
            for(int j = 1; j <= i; j++){
                System.out.print(i * j + " ");
            }
            System.out.println();
        }
    }
}

File: C#/ForEncadeado.cs
using System;

public class ForEncadeado{
    static void Main(string[] args){

        Console.WriteLine("Digite um numero inteiro: ");
        int x = Convert.ToInt32(Console.ReadLine());

        for(int i = 1; i <= x; i++){
            for(int j = 1; j <= i; j++){
                Console.Write(i * j + " ");
            }
        Console.WriteLine();
        }
    }
}